client/src/FIleToolbar.js

```diff
@@ -0,0 +1,321 @@
+        </Button>
+        <Button
+          color="light"
+          onClick={copy}
+          size="sm"
+          title="Copy"
+        >
+          <i className="fa fa-copy" />
+        </Button>
+        <Button
+          color="light"
+          onClick={paste}
+          size="sm"
+          title="Paste"
+        >
+          <i className="fa fa-paste" />
+        </Button>
+      </ButtonGroup>
+    </div>
+  );
+}
+
+function SelectionButtons() {
+  const { toggleAllSelectedFiles } = useContext(FileManagerContext);
+  return (
+    <div className={classNames('d-flex', 'flex-column', 'justify-content-center', 'align-items-center', styles.ButtonGroupWrapper)}>
+      <p className={styles.ButtonGroupTitle}>Select</p>
+      <ButtonGroup className={styles.ButtonGroup}>
+        <Button
+          color="light"
+          onClick={toggleAllSelectedFiles}
+          size="sm"
+          title="(Un)Select all"
+        >
+          <i className="fa fa-asterisk" />
+        </Button>
+      </ButtonGroup>
+    </div>
+  );
+}
+
+function OperationsButtons() {
+  const { del, edit, isFileSelected, rename, view } = useContext(FileManagerContext);
+  return (
+    <div className={classNames('d-flex', 'flex-column', 'justify-content-center', 'align-items-center', styles.ButtonGroupWrapper)}>
+      <p className={styles.ButtonGroupTitle}>View/edit</p>
+      <ButtonGroup className={styles.ButtonGroup}>
+        <Button
+          color="light"
+          disabled={!isFileSelected}
+          onClick={view}
+          size="sm"
+          title="View file"
+        >
+          <i className="fa fa-eye" />
+        </Button>
+        <Button
+          color="light"
+          disabled={!isFileSelected}
+          onClick={edit}
+          size="sm"
+          title="Edit file"
+        >
+          <i className="fa fa-edit" />
+        </Button>
+        <Button
+          className={styles.iconRename}
+          color="light"
+          onClick={rename}
+          size="sm"
+          title="Rename"
+        >
+          <i className="icon-rename" />
+        </Button>
+        <Button
+          color="light"
+          onClick={del}
+          size="sm"
+          title="Delete"
+        >
+          <i className="fa fa-trash" />
+        </Button>
+      </ButtonGroup>
+    </div>
+  );
+}
+
+function TarballButtons() {
+  const { extract, pack } = useContext(FileManagerContext);
+  return (
+    <div className={classNames('d-flex', 'flex-column', 'justify-content-center', 'align-items-center', styles.ButtonGroupWrapper)}>
+      <p className={styles.ButtonGroupTitle}>Extract</p>
+      <ButtonGroup className={styles.ButtonGroup}>
+        <Button
+          color="light"
+          onClick={pack}
+          size="sm"
+          title="Pack"
+        >
+          <i className="fa fa-compress" />
+        </Button>
+        <Button
+          color="light"
+          // XXX Better disability.
+          // disabled={!isFileSelected}
```

Comment:
Unused code to remove

Reply:
Other unused code are removed but this one looks like a legacy comment left by the previous development so I would temporarily keep it for a bit longer.